src/resources/views/edit.blade.php

```diff
@@ -16,16 +17,17 @@
 
 @section('content')
 <div class="row">
-	<div class="col-md-8 col-md-offset-2">
+	<div @if (!$request->ajax())class="col-md-8 col-md-offset-2"@endif>
 		<!-- Default box -->
-		@if ($crud->hasAccess('list'))
+		@if ($crud->hasAccess('list') AND !$request->ajax())
```

Comment:
What's that `AND`?

Reply:
It is a bit redundant to display a link to the list inside the form. So it is hidden inside the ajax form.

Reply:
I meant why `AND` vs `&&`?